website/docs/reference/widgets/list.md

```diff
@@ -41,70 +35,110 @@ You can display static JSON data in the `Items` property for generating the list
 
 Follow the steps below to paginate the responses and request smaller chunks of data at a time:
 
-1. Enable the Server Side Pagination property for the List.
-2. Call the query on the onPageChange event listener.
-3. Set the LIMIT and OFFSET clause in the query:
+1. Enable the **Server Side Pagination** property for the List.
+2. Call the query on the **onPageChange** event listener.
+3. Set the `LIMIT` and `OFFSET` clause in the query:
 
 ```javascript
 SELECT * FROM users LIMIT {{ List1.pageSize }} OFFSET {{ (List1.pageNo - 1) * List1.pageSize }}
 ```
 
 ## Access list items
 
-You can reference the values inside each item selected in the list using the `selectedItem` property as shown below:
+You can reference the values inside each item selected in the List using the **selectedItem** property as shown below:
 
 ```javascript
 
 {{listName.selectedItem.fieldName}}
 ```
 
-**Example**: in the preceding example, if you want to display the book name of the selected item in a Text widget of `List1`, bind it in the **Text** property of the Text widget as shown below 
+**Example**:In the preceding example, if you want to display the book name of the selected item in a Text widget of `List1`, bind it in the **Text** property of the Text widget as shown below 
 
 ```javascript
 
 {{List1.selectedItem.bookName}}
 ```
 
-To access the index of the selected item in the list, you can use the following:
+To access the index of the selected item in the List, you can use the **selectedItemIndex** as shown below:
 
 ```javascript
 
 {{listName.selectedItemIndex}}
 ```
 
+You can access the sibling widgets inside a list item card using the **currentView** property. 
+
+**Example:**If you have an Input widget and a Button widget inside the List and want to use the Input's `Text` property to show an alert message on button click. In the button widget's *onClick* event, you can access the input widgetas shown below
+
+```javascript
+{{showAlert(currentView.Input1.text)}}
+```
+The **currentView** property should always be used to access sibling data instead of referencing it directly Eg: `{{Input1.text}` may seem to work in the app's Edit mode but won't work when deployed.                                                    
+
+## Nested lists
+
+You can nest lists within a List widget up to three levels. The **level_*** property can be used to access the parent List item's data and widget properties where * represents the level number (from 1 through 3).
+
+Suppose there is a parent list - `List1`, and a child list - `List2`. The widgets present in the inner list List2 can access the values of an attribute/field in the dataset using the **currentItem** property of the outer list List1 as shown below:
```

Comment:
Let's not use names as `List1` and `List2` let's use `ParentList` and `ChildList`

Reply:
Since there are 3 let's name them as outer, middle and inner. Please let's avoid using bad names in our docs, as this leads to people also declaring bad names for their widgets.

Reply:
Updated